### PR TITLE
feat(icons): implement new autofocus icon

### LINKED ISSUE #261: Add new Autofocus icon
### Describe the feature

Add new icon AltAutofocusIcon.

### Additional context

Screenshot from 2023-03-24 15-51-49

### PR DESCRIPTION
**Pull request by Sana-Shana**
### Describe your changes

Add a new AltAutofocus icon.

### Issue ticket number and link

- Fixes #261 

### Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have verified that the code builds perfectly fine on my local system
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have verified that my code follows the style already available in the repository
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/icons/src/autofocus.svg [hunk 1]
```diff
@@ -1,9 +1,3 @@
...
-  Based on Material design icons by Google - https://github.com/google/material-design-icons
-  which are licensed under the Apache License 2.0. Icon 'crop_free (filled)'.
-
-  File is modified. Modifications are licensed under the MIT license.
--->
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-  <path d="M5 15H3v4a2 2 0 002 2h4v-2H5zM5 5h4V3H5a2 2 0 00-2 2v4h2zM19 19h-4v2h4a2 2 0 002-2v-4h-2zM19 3h-4v2h4v4h2V5a2 2 0 00-2-2zM12.77 7.17h-1.52l-3.2 8.6h1.47l.84-2.38h3.28l.85 2.38H16zm-2 5l.77-2.17A12.77 12.77 0 0012 8.38c0 .15.11.4.21.74s.19.58.24.73l.81 2.34z" />
-</svg>
+    <path d="M15.334 22v-2.222h4.444v-4.445H22v4.445A2.228 2.228 0 0 1 19.779 22Zm-11.11 0A2.23 2.23 0 0 1 2 19.778v-4.445h2.223v4.445h4.444V22Zm9.562-6.966V8.671h4.362v1.076h-3.077v1.506h2.656v1.077h-2.656v2.7Zm-2.354 0-.556-1.445H8.333l-.525 1.445H6.445l2.478-6.363h1.359l2.547 6.363Zm-2.7-2.518h1.736l-.877-2.361Zm11.05-3.849V4.222h-4.444V2h4.445A2.228 2.228 0 0 1 22 4.222v4.445Zm-17.778 0V4.222A2.23 2.23 0 0 1 4.224 2h4.443v2.222H4.223v4.445Z"/>
+</svg>
```
Comment (boilund) [inline packages/icons/src/autofocus.svg:3]: No new line?

File: packages/icons/src/autofocus.svg [hunk 2]
```diff
@@ -1,9 +1,3 @@
-<!-- 
-  Based on Material design icons by Google - https://github.com/google/material-design-icons
-  which are licensed under the Apache License 2.0. Icon 'crop_free (filled)'.
-
-  File is modified. Modifications are licensed under the MIT license.
--->
```
Comment (boilund) [inline packages/icons/src/autofocus.svg:6]: Isn't it based on the Material icon?

Comment (Sana-Shana) [reply packages/icons/src/autofocus.svg]: I missed some parts and just updated the commit.

File: packages/icons/src/autofocus.svg (hunk 1 above)
Comment (Sana-Shana) [reply packages/icons/src/autofocus.svg]: I missed some parts and just updated and fixed the issues.

**Comment by Sana-Shana** [comment]
> Maybe the commit header is not correct? You can just write "feat(icons): add AltAutofocus icon".

Fixed!

**Review by boilund — APPROVED** [review]

**Pull request merged**